assets/javascripts/modules/addRemove.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * Add/Remove Module
+ *
+ * Usage:
+ *
+ *  <div class="add-remove"
+ *        data-add-remove
+ *        data-add-text="Add another redirect URI"
+ *        data-attr-name="redirectUris[$]"
+ *        data-attr-id="redirectUris[$]">
+ *
+ *   <script type="text/template" data-add-remove-template>
+ *     <li class="add-remove--item" data-add-remove-item>
+ *       <input type="text" value="" data-add-remove-unique/>
+ *     </li>
+ *   </script>
+ *
+ *    <ul data-add-remove-list>
+ *      <li data-add-remove-item>
+ *        <input type="text" value="Value 1" name="redirectUris[0]" id="redirectUris[0]" data-add-remove-unique/>
+ *      </li>
+ *      <li data-add-remove-item>
+ *        <input type="text" value="Value 2" name="redirectUris[1]" id="redirectUris[1]" data-add-remove-unique/>
+ *      </li>
+ *      <li data-add-remove-item>
+ *        <input type="text" value="Value 3" name="redirectUris[2]" id="redirectUris[2]" data-add-remove-unique/>
+ *      </li>
+ *    </ul>
+ *
+ *  </div>
+ *
+ */
+
+
+var addRemoveContainer = '[data-add-remove]',
+    addRemoveList      = '[data-add-remove-list]',
+    addRemoveItem      = '[data-add-remove-item]',
+    addButton          = '[data-add-btn]',
+    removeButton       = '[data-remove-btn]',
+    maxItems           = 'data-add-remove-max';
+
+
+/**
+ * Init add/remove, where necessary
+ *
+ * @param
+ */
+module.exports = function() {
+
+  var $addRemoveContainers = $(addRemoveContainer);
+
+  // only continue if counter html exists
+  if($addRemoveContainers.length === 0) return;
+
+  // for each add/remove container on the page
+  $addRemoveContainers.each(function() {
+
+    var $container = $(this);
+
+    $container.find(addRemoveItem).each(function() {
+
+      // only insert Remove button if removable is set
+      if($(this).is('[data-removable]')) {
```

Comment:
the find on line 60 - can it be changed to `[data-removable]` as the conditional on line 63 only has one path/direction